src/ontology/cl-edit.owl

```diff
@@ -24453,6 +24454,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_0019032 "intestinal tuft cell"@en)
 EquivalentClasses(obo:CL_0019032 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001277)))
 
+# Class: obo:CL_0300023 (hillock cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7772-9578>) Annotation(oboInOwl:hasDbXref "PMID:30069044"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32726565"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:33442032"^^xsd:string) obo:IAO_0000115 obo:CL_0300023 "An epithelial, transitional cell type between basal and secretory; located in stratified, non-ciliated structures with high cell turnover. In some mammalian species, this cell type has been noted to express KRT13 and KRT4 and is postulated to play a role in squamous barrier function and immunomodulation."^^xsd:string)
```

Comment:
Definition should mention location in respiratory epithelium

Reply:
hillock cell definition should **not** be restricted to respiratory epithelium because there is evidence of the same cell type in urethra from https://doi.org/10.1016/j.celrep.2018.11.086
Perhaps instead hillock cell of respiratory epithelium, hillock cell with these 3 hillock cell types should be added: hillock cell of prostatic urethra and hillock cell of proximal prostatic duct?
Evidence:
"These data demonstrate a strong correlation between SCGB1A1+ prostate epithelia and lung Scgb1a1+ club cells, as well as lung Krt13+ hillock cells. KRT5+/KRT13+ prostate epithelia display a strong positive correlation with lung Krt5+ basal and Krt13+ hillock cells. Club and Hillock Epithelial Cell Types Are Enriched in the Urethra and Peri-urethral Prostate Zones KRT5+/KRT14−/KRT13+ hillock epithelial cells are abundant in the prostatic urethra and collecting ducts, as well as the central zone surrounding the [ejaculatory ducts](https://www.sciencedirect.com/topics/immunology-and-microbiology/ejaculatory-duct), but are rare in the peripheral zone ([Figure 5](https://www.sciencedirect.com/science/article/pii/S2211124718318771#fig5)E)."
and 
"Prostate KRT13+ cells are similar in morphology and transcriptomic profile to hillock [basal cells](https://www.sciencedirect.com/topics/immunology-and-microbiology/basal-cell) of the lung ([Montoro et al., 2018](https://www.sciencedirect.com/science/article/pii/S2211124718318771#bib36)). Prostate hillock cells are also concentrated in the prostatic urethra and proximal prostatic ducts ([Figure 5](https://www.sciencedirect.com/science/article/pii/S2211124718318771#fig5))"

Reply:
PS if the cells are "defined" by the markers, consider adding express some PR:XXXXXXX axioms (assuming they are proteins, or at least a proxy to proteins)